src/main/java/com/snack/news/util/SlackAuthHttpClient.java

```diff
@@ -0,0 +1,59 @@
+package com.snack.news.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.snack.news.domain.slack.SlackChannel;
+import org.springframework.stereotype.Component;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Map;
+
+@Component
+public class SlackAuthHttpClient {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'SlackAuthHttpClient' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=5471424)